app/src/features/apiClient/screens/apiClient/components/views/http/HttpClientView.tsx

```
@@ -818,13 +818,15 @@ const HttpClientView: React.FC<Props> = ({
 
 const WithQueryParamsProvider = (Component: React.ComponentType<any>) => {
   return (props: any) => {
+    const record = useAPIRecordsStore().getState().getData(props.apiEntryDetails.id) as RQAPI.ApiClientRecord;
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)